### ISSUE TITLE
notebook Google Colab

### ISSUE THREAD

**Issue by HobisPL** [OlaWod/FreeVC#43]
Can you create a training model notebook on Google Colab?
I tried to do it, but something is not working.

**Comment by OlaWod** [comment]
I'm afraid it is not practical to train on Colab, as Google Drive only has 15 GB free space. 
```bash
# set the colab device to GPU
!git clone https://github.com/OlaWod/FreeVC.git
cd FreeVC
# do the data preprocessing according to your need
!python train.py -c configs/freevc.json -m freevc
```

**Comment by HobisPL** [comment]
Google Colab has around 60GB, Google Drive has 15GB.
I mainly train other technologies from Nvidia (Tacotron 2, Flowtron, Rad-TTS) and Diff-SVC on Google Colab. I wanted to see how you would perform on the Polish language.

**Comment by OlaWod** [comment]
To be honest I feel a bit lazy to create a colab notebook. Aren't it feasible to just git clone this repo, and run the commands in the README with a "!" symbol in front of the command in colab?
And btw the default SR augmentation will create 24x original data size.
And if you only want to do inference I've offered a demo in [huggingface](https://huggingface.co/spaces/OlaWod/FreeVC).

**Comment by steven850** [comment]
Yeah 60GB wouldn't cut it.  VCTK with SR comes to 630GB after preprocess plus the Wavlm model etc... you need at least 700GB to train.

**Comment by rbychn** [comment]
> And if you only want to do inference I've offered a demo in [huggingface](https://huggingface.co/spaces/OlaWod/FreeVC).

Could you please add a wiki or readme on how to get that UI on HuggingFace working locally? Would be great if you could also add UI for training in there so everything can be done locally, easily.

**Comment by OlaWod** [comment]
> > And if you only want to do inference I've offered a demo in [huggingface](https://huggingface.co/spaces/OlaWod/FreeVC).
> 
> Could you please add a wiki or readme on how to get that UI on HuggingFace working locally? Would be great if you could also add UI for training in there so everything can be done locally, easily.

for inference you can clone the huggingface repo and run `app.py` and go to `http://localhost:7860/` in the browser. ([gradio quickstart fyi](https://gradio.app/quickstart/))
for training, sorry i'm lazy to do that.
